source/standards/publish-opensource-code.html.md.erb

```diff
@@ -9,22 +9,22 @@ expires: 2018-09-01
 
 Wherever possible, we [make our source code open and reusable][]. This means other government departments and people in outside organisations can benefit from our work. We also maintain several open source projects developed for use on GOV.UK and with other work we do, such as the [Puppet module for go-audit][] or the [GDS Operations open source web site][].
 
-It’s not always appropriate to open code, there are situations [when you should keep some data and code closed][], for example when involving keys and security credentials.
+It’s not always appropriate to open code. There are situations [when you should keep some data and code closed][], for example when involving keys and security credentials.
 
 The Service Manual explains [how to open previously closed code and your responsibilities for maintaining open code][].
 
-You should release any open code with a clear license for reuse. As our documentation repositories include both content and code, we use 2 licences:
+You should release any open code with a clear license for reuse. As repositories often include both documentation and code, we use 2 licences:
 
 * [MIT licence][]  for code and code samples in documentation
 * [Open Government Licence (OGL)][] for documentation
 
 When you publish open source code, your project must:
 
-* include a readme, for example the [Digital Marketplace API][]
+* include a readme, for example [READMEs for GOV.UK applications][]
```

Comment:
You've written "for example", but that's not an example. Perhaps something like:

```markdown
include a readme. You could follow [GOV.UK's guidance for writing a README][]
```

would be better?